ngraph/core/include/ngraph/op/util/sub_graph_base.hpp

```diff
@@ -345,6 +345,10 @@ namespace ngraph
     }
     template class NGRAPH_API FactoryRegistry<op::util::SubGraphOp::InputDescription>;
 
+    template <>
+    FactoryRegistry<op::util::SubGraphOp::InputDescription>&
```

Comment:
Why do we need this specialization?

Reply:
It's an unresolved symbol when compiled statically.